### ISSUE TITLE
Check site accessibility

### ISSUE THREAD

**Issue by tildeeine** [tildeeine/tildeeine.github.io#56]
Check that site is accessible for color blind, check guidelines to see what more should be done for accessibility

**Comment by tildeeine** [comment]
Fix issues reported in [wave](https://wave.webaim.org/report#/tildeeine.github.io).
